CMakeLists.txt

```diff
@@ -240,6 +241,10 @@ if(BUILD_TESTING)
     manifest_parser_perftest
   )
     add_executable(${perftest} src/${perftest}.cc)
+    set_source_files_properties(src/${perftest}.cc
+      PROPERTIES
+        COMPILE_DEFINITIONS NINJA_PYTHON="${NINJA_PYTHON}"
```

Comment:
What if `NINJA_PYTHON` is empty, because it not found by `find_package(Python3 COMPONENTS Interpreter)` (and therefore `Python3_EXECUTABLE` is empty)?

Reply:
yea i think it would fail similarly to today, I can make it error in cmake if you'd like?